.pre-commit-config.yaml

```
@@ -407,19 +407,29 @@ repos:
       - id: language-matters
         language: pygrep
         name: Check for language that we do not accept as community
-        description: Please use "deny_list" or "allow_list"  instead.
-        entry: "(?i)(black|white)[_-]?list"
+        description: Please use more appropriate words for community documentation.
+        entry: >
+          (?ix)
+          (black|white)[_-]?list|
+          \bmaster\b|
+          \bhe\b|
+          \bshe\b|
+          \bhis\b|
+          \bher\b|
+          \bslave\b|
+          \bsanity\b|
+          \bdummy\b|
         pass_filenames: true
         exclude: >
           (?x)
           ^airflow/www/fab_security/manager\.py$|
           ^airflow/providers/apache/cassandra/hooks/cassandra\.py$|
           ^airflow/providers/apache/hive/operators/hive_stats\.py$|
-          ^airflow/providers/apache/hive/.*PROVIDER_CHANGES_*|
-          ^airflow/providers/apache/hive/.*README\.md$|
           ^tests/providers/apache/cassandra/hooks/test_cassandra\.py$|
           ^docs/apache-airflow-providers-apache-cassandra/connections/cassandra\.rst$|
           ^docs/apache-airflow-providers-apache-hive/commits\.rst$|
+          ^.pre-commit-config\.yaml$|
+          ^.*CHANGELOG\.txt$|CHANGELOG\.rst$
```

I think you are missing `|` here ?
```suggestion
          ^.*CHANGELOG\.txt$|CHANGELOG\.rst$|
```